### PR TITLE
Get song logs from db instead of redux store

### PR DESCRIPTION
**Pull request by veronicz**
- UserFeed and SongLog components subscribe to UserSongs publication instead of getting tracks from the redux store
- HideSongButton component gets 'show' property directly from Song component, instead of a separate db subscription
- Removed songLogReducer as it is no longer needed
- Added tunguska:reactive-aggregate to publish aggregation
- Some other code refractoring

### CODE REVIEW

**Comment by Talos6** [comment]
return (dispatch, getState) => {
share we still keep this kind of format cause we didn't use any dispatch or get State

**Comment by veronicz** [comment]
> return (dispatch, getState) => {
> share we still keep this kind of format cause we didn't use any dispatch or get State

**Comment by veronicz** [comment]
> return (dispatch, getState) => {
> share we still keep this kind of format cause we didn't use any dispatch or get State

Yea I think so, otherwise we can only return a JSON object. To use Meteor.call or anything async, we need a middleware like redux-thunk. But maybe getState is not necessary.

File: sound-bubble/imports/ui/components/Account/UserFeed.jsx [hunk 1]
```diff
@@ -11,8 +14,9 @@ class UserFeed extends Component {
 
   getSongDetails() {
     const { myRecentTracks } = this.props;
+    console.log('myrecent', myRecentTracks);
```
Comment (martipal) [inline sound-bubble/imports/ui/components/Account/UserFeed.jsx]: Still need this?

File: sound-bubble/imports/ui/components/Home/SongLog.jsx [hunk 2]
```diff
@@ -1,23 +1,18 @@
...
-      .map(t => {
-        return (
-          <Song
-            key={t.songId + t.userId + t.timestamps}
-            track={t}
-            home={true}
-          />
-        );
-      });
+    console.log('groupRecent', this.props.groupRecentTracks);
```
Comment (martipal) [inline sound-bubble/imports/ui/components/Home/SongLog.jsx]: still need?

File: sound-bubble/server/methods/accountMethods.js [hunk 3]
```diff
@@ -4,36 +4,23 @@ import UserSongs from '../../imports/api/userSongs';
...
-        }
+function updateHideStatus(songId, option) {
+  UserSongs.update(
+    {
+      userId: Meteor.user().profile.id,
+      songId: songId
+    },
+    {
+      $set: {
+        show: option === 'show'
```
Comment (martipal) [inline sound-bubble/server/methods/accountMethods.js]: nice

File: sound-bubble/imports/ui/components/Home/SongLog.jsx (hunk 2 above)
Comment (veronicz) [reply sound-bubble/imports/ui/components/Home/SongLog.jsx]: I kinda want to keep this as song logs are buggy sometimes. Yancey and I have seen users in one group get the same, duplicated song logs from one user (happened in our last demo, too).

I think it's something to do with the tokens but I don't understand how that happened. I would want to keep this console log until groups and the load more button are fully implemented.

File: sound-bubble/imports/ui/components/Account/UserFeed.jsx (hunk 1 above)
Comment (veronicz) [reply sound-bubble/imports/ui/components/Account/UserFeed.jsx]: This too

**Pull request merged**